plugins/scaffolder-backend/sample-templates/react-ssr-template/template.yaml

```diff
@@ -4,7 +4,9 @@ metadata:
   name: react-ssr-template
   title: React SSR Template
   description: Next.js application skeleton for creating isomorphic web applications.
+  annotations:
+    tags: Recommended,React
```

Comment:
So this becomes an open question - how do we want to store tags? @freben @benjdlambert

Reply:
I’m favour of using yaml and storing a list rather than splitting strings.

```yml
tags:
  - experimental
  - golden
  
```

Maybe these could be references to another tag type in the entity catalog?

Reply:
Let's decide if we want to figure it out or can do it in iterative way? 
As a temp thing I can remove tags until we figure out the implementation

Reply:
@benjaminr @Rugvip @freben

Reply:
I think just having a list is fine for now, it's part of the architecture weekly tomorrow so we can have a chat and decide what we want to do